MonMaraicher/DetailFarmerView.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  DetailFarmerView.swift
+//  MonMaraicher
+//
+//  Created by Yves Charpentier on 27/02/2024.
+//
+
+import SwiftUI
+import MapKit
+
+struct DetailFarmerView: View {
```

Comment:
In english to say "les details du maraicher/fermier" -> "FarmerDetails" in that order and with "s"